app/assets/stylesheets/_colours.scss

```diff
@@ -0,0 +1,78 @@
+$black:         #000;
```

Comment:
Cool. Only issue now is that it looks like the variables in agency aren't the same